test/suite/suite_subprocess.py

```diff
@@ -206,6 +206,14 @@ def run_subprocess_function(self, directory, funcname):
         returncode = -1
         os.makedirs(directory)
 
+        # This hack fixes test_bug018. We set the LD_PRELOAD before opening another python instance.
```

Comment:
I would like to direct your attention to this hack, should we have a hook?

Reply:
Hooks are meant to insert additional behaviour so it feels slightly inaccurate to use a hook. Do we have many tests? I'd be happy to start with a 
```
if os.environ.get("TESTUTIL_TSAN") == "1":
    self.skipTest("Not compatibable with TSan")
```
if there's just one or two

Reply:
Done.